### PR TITLE
Accueil page

### PR DESCRIPTION
**Pull request by ozen2**

### CODE REVIEW

File: client/src/components/NavBar.jsx
Comment (FabienRollet) [inline client/src/components/NavBar.jsx:1]: English here pls

File: client/src/components/ReservationModule.jsx [hunk 1]
```diff
@@ -45,9 +46,9 @@ export default function ReservationModule() {
         value={formData.selectedTripType}
       />
       <DropdownTraveller />
-      <button className="reservationButton" type="submit">
+      <Link to="/booking" className="reservationButton" aria-label="réserver">
         Réserver
```
Comment (FabienRollet) [inline client/src/components/ReservationModule.jsx]: same, english pls

File: client/src/pages/Home.jsx [hunk 2]
```diff
@@ -77,6 +74,7 @@ function Home() {
         imageLeft={thumbnail1.url}
         imageMid={thumbnailUrl2.url}
         imageRight={thumbnailUrl3.url}
+        descriptionCards="Discover the 3 Humans' Favorite Planets"
       />
       <HomeCards
         sectionTitle="Proches de vous"
```
Comment (FabienRollet) [inline client/src/pages/Home.jsx]: same 👀

File: client/src/styles/ReservationModule.scss [hunk 3]
```diff
@@ -16,10 +25,15 @@ form {
...
-      position: absolute;
-      top: 36rem;
-      right: 1rem;
+      margin-right: 1rem;
+      justify-self: center;
+      grid-column: 2/4;
+      transition: all ease-in-out 0.3s;
+    }
+    .reservationButton:hover {
+      background-color: #5907c4;
```
Comment (FabienRollet) [inline client/src/styles/ReservationModule.scss]: no variable :(
(also line 9 but it looks specific so idk)

File: client/src/styles/Slogan.scss [hunk 4]
```diff
@@ -1,38 +1,33 @@
...
   }
   h3:first-of-type {
     color: $whiteColor;
-    text-shadow: 0 0 10px #9548f9cd;
+    text-shadow: 0 0 10px #000000;
   }
   h3:nth-child(3) {
     color: $whiteColor;
-    text-shadow: 0 0 10px #9548f9cd;
+    text-shadow: 0 0 10px #000000;
```
Comment (Gamarin0) [inline client/src/styles/Slogan.scss:17]: $whitecolor look prety much the same here ?

Comment (FabienRollet) [inline client/src/styles/Slogan.scss:17]: $whitecolor prety much looks the same here no ?

**Review by FabienRollet — CHANGES_REQUESTED** [review]

**Review by KevinDavoust — APPROVED** [review]

**Review by FabienRollet — APPROVED** [review]

**Pull request merged**